.github/workflows/ci.yaml

```diff
@@ -59,10 +55,60 @@ jobs:
         run: yarn test:unit
         if: success()
 
+  codecov:
+    name: Upload coverage report to Codecov
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+
+      - name: Install Node.js v12
+        uses: actions/setup-node@v2
+        with:
+          node-version: "12"
+
+      - name: Fetch dependencies from cache
+        id: cache-yarn
+        uses: actions/cache@v2
+        with:
+          path: "**/node_modules"
+          key: yarn-build-${{ hashFiles('**/yarn.lock') }}
+
+      - name: Install dependencies
+        if: steps.cache-yarn.outputs.cache-hit != 'true'
+        run: yarn --frozen-lockfile
+
```

Comment:
The codecov upload relies on the coverage being generated, which happens when we run the unit tests. So if this is a fresh checkout of the repo, you may need to add a step after install dependencies and before upload code coverage to run the unit tests.

Reply:
Ah, I'll revert that change if it depends on the unit tests in the first place.

Reply:
Feel free to add a comment too (apologies, I should have done that when I added it!)

Reply:
In hindsight it makes perfect sense that you need to run tests to know what coverage for the tests is like - I just didn't notice in a hurry, heh. Pushed before I could add a comment, fingers crossed its fine.